beacon-chain/sync/subscriber_beacon_blocks_chunks.go

```diff
@@ -169,7 +170,21 @@ func (s *Service) reconstructBlockFromChunk(ctx context.Context, chunk interface
 		logrus.WithError(err).Error("Could not decode block data")
 		return
 	}
-	if err := s.beaconBlockSubscriber(ctx, msg); err != nil {
+	// We overwrite the signature in the block to keep it to be the original one in the database
+	// Unfortunately to do this without reflection we make extra copies of the full block. We could switch over the type instead.
```

Comment:
This is unfortunate, depending on the cost here we may want to just overwrite directly on msg but not clear what's the best way